common/app/views/fragments/media/audio.scala.html

```diff
@@ -2,13 +2,15 @@
 
 @trackingCode(target: String) = {@if(player.audio.tags.isPodcast){podcast:}@target:@player.title}
 
+@acastUrl(url: String) = @{ if (player.audio.tags.isPodcast) "https://flex.acast.com/" + url.replace("https://", "") else url }
```

Comment:
I think it would be better to add this logic here: https://github.com/guardian/frontend/blob/e39e723c3cde08a5f9b278e33d02594ba7eae58b/common/app/model/content.scala#L558
so it can be tested for example.

Reply:
Agree, good to keep logic out of templates (even though we admittedly have lots already in the templates).

Reply:
not sure this is the best strategy especially as it is [not encouraged by the framework and the templating language](https://www.playframework.com/documentation/2.6.x/ScalaTemplates), but anyway done it.